XRTK-Core/Packages/com.xrtk.core/Interfaces/Providers/Controllers/IMixedRealityInputDataProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace XRTK.Interfaces.Providers.Controllers
+{
+    /// <summary>
+    /// Mixed Reality Toolkit data provider definition, used to instantiate and manage devices, controllers and input for specific SDKs and libraries.
+    /// </summary>
+    public interface IMixedRealityInputDataProvider : IMixedRealityDataProvider
```

Comment:
break this out into a new PR along with the updates to the dictation and speech data providers

Reply:
See #509